src/dev/flang/be/interpreter/Intrinsics.java

```diff
@@ -273,7 +273,7 @@ public static Callable call(Interpreter interpreter, Clazz innerClazz)
           Path path = Path.of(new String(pathBytes, StandardCharsets.UTF_8));
           try
             {
-              boolean b = Files.deleteIfExists(path);
+              boolean b = Files.deleteIfExists(path); // NYI : not sure whether to keep it or use Files.delete instead because the existence check already takes place in Fuzion 
```

Comment:
Even if there is an existence test before, another process might have created the file at any time.  So the question is what is the behaviour of `Files.delete` if it does not exist. If this would throw an `IOException`, this would not be good since then we could see sporadic exceptions for a file that is created by a concurrent process, `deleteIfExists` might be better.